<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_LessonChange</class>
 <widget class="QWidget" name="Widget_LessonChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox_Lesson">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>可修改的课程</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>可更新的字段（可多选）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>69</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>课程名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>星期数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>节数</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_week">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_location">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_index">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>210</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>地点</string>
   </property>
   <widget class="QRadioButton" name="radio_xitucheng">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>西土城校区</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_shahe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>沙河校区</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>69</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>课程群号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_qun">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_extra">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
